### DISCUSSION TITLE
TRUST4 on snRNAseq

### DISCUSSION THREAD

**Discussion by sinanassiri** [liulab-dfci/TRUST4#55] [Q&A]
Dear TRUST4 team,

I was wondering if TRUST4 can be used with 10X 5' single-nucleus RNAseq data. Based on your experience, is TCR assembly from snRNAseq (with or without additional VDJ enrichment) even feasible? Would greatly appreciate if you can share your experience.

Thanks,
Sina

**Comment by mourisl** [comment]
We've never tested on single-nucleus RNAseq data before. Theoretically, TRUST4 can handle 5' snRNAseq without VDJ enrichment.
